perfrunbook/debug_hw_perf.md

```diff
@@ -136,3 +136,44 @@ Backend stalls are caused when the CPU is unable to make forward progress execut
 6. If back-end stalls due to the cache-system and memory system are the problem, the data-set size and layout needs to be optimized.
 7. Proceed to [Section 6](./optimization_recommendation.md) to view optimization recommendations for working with a large data-set causing backend stalls.
 
+### Capturing Coherent Mesh Network hardware event counters
```

Comment:
Move this to the appendix.md section where we have the more advanced topics like eBPF usage.  The primary difficulty with the Coherent Mesh counters is their lack of attribution to any specific code event.

Reply:
done